tutorials/rise_timeseries_frb.ipynb

```diff
@@ -0,0 +1,506 @@
+{
```

Comment:
<p>I would put the paper name and an underlying link. </p><p>Also would add link to the RISE paper (just linked to the RISE abbreviation): <a href="http://bmvc2018.org/contents/papers/1064.pdf" target="_blank">http://bmvc2018.org/contents/papers/1064.pdf</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>The linked paper is the RISE paper, this part of the introduction is actually the same for all rise tutorials we have. I'll make a new issue to fix it in all of them.</p><p> </p>

Reply:
See issue #724